integration-test/pipeline/rest-integration.js

```diff
@@ -27,8 +27,16 @@ export function CheckIntegrations() {
     });
 
     var id = "github";
-    var cdef = http.request("GET", `${pipelinePublicHost}/v1beta/connector-definitions/${id}`, null, null).
-      json().connectorDefinition;
+    var cdefs = http.request("GET", `${pipelinePublicHost}/v1beta/component-definitions?pageSize=100`, null, null).
```

Comment:
👍 I absolutely overlooked this, it should have been `component-definitions` since the beginning.

Instead of looping over the results, you can use the `filter` query param:

```suggestion
    var cdefs = http.request("GET", `${pipelinePublicHost}/v1beta/component-definitions?filter=qTitle="GitHub"`, null, null).
```

Reply:
Yes, that's better. Updated.